pkg/apis/componentconfig/types.go

```diff
@@ -77,6 +77,9 @@ type ControllerManagerConfiguration struct {
 	OSBAPIContextProfile   bool
 	OSBAPIPreferredVersion string
 
+	// OSBAPITimeOut is the interval on the Open Service Broker API spec.
```

Comment:
I don't understand what this comment says.

Reply:
@MHBauer It means the length of the timeout of any request to the broker, in seconds. I am sorry and I will modify this comment.